### ISSUE TITLE
shortLinkResolveTimeOut deep link config option

### ISSUE THREAD

**Issue by toddtreece** [singular-labs/react-native-sdk#12]
we ran into an issue where shortened singular links are not being sent to the deep link handler. although i couldn't find documentation, it seems like the `shortLinkResolveTimeOut` config option fixes the issue:
https://github.com/singular-labs/react-native-sdk/blob/523ebb91223dbe2928eb2ae7bc21bff38195e4ba/ios/SingularBridge.m#L64

the `objectForKey:@"shortlinkResolveTimeout"` option passed from JS is expected to have a slightly different capitalization which was very confusing and hard to spot, especially because it's not exposed in the `SingularConfig` class properties:
https://github.com/singular-labs/react-native-sdk/blob/523ebb91223dbe2928eb2ae7bc21bff38195e4ba/SingularConfig.js#L2-L12

a related issue seems to be that shortened singular deep links do not work when the app is backgrounded, but the same links work fine if a deep link is passed in via query string. i couldn't find a way to pass in the `shortLinkResolveTimeOut` in that situation.

**Comment by tplorts** [comment]
This is issue can probably be closed since I see that `shortlinkResolveTimeout` is not used anymore, at least in the react-native sdk.

I am however curious to know if you have any more info on this problem.
In our usage of this sdk, we had set `shortlinkResolveTimeout` to 5 seconds, and we think that fixed the problem at the time.
Now that `shortlinkResolveTimeout` hasn't been used by this sdk for months, I'm assuming that means the underlying problem was fixed - is that so?
